qhub/deploy.py

```diff
@@ -111,3 +113,12 @@ def guided_install(
 
     # 06 Full deploy QHub
     terraform.apply(directory="infrastructure")
+
+
+def add_clearml_dns(zone_name, record_name, record_type, ip_or_hostname):
+    logger.info(f"Setting DNS record for ClearML for record: {record_name}")
+    update_record(zone_name, f"clearml.app.{record_name}", record_type, ip_or_hostname)
+    update_record(zone_name, f"clearml.api.{record_name}", record_type, ip_or_hostname)
+    update_record(
+        zone_name, f"clearml.files.{record_name}", record_type, ip_or_hostname
+    )
```

Comment:
This seems like a really interesting and awesome integration with qhub, it piqued my interest enough to take a look at this PR to see how it is being integrated.

One thing I noticed, and I may be missing something here, but the docs in this PR indicate the domain as `app.clearml.your-qhub-domain.com`, which seems appropriately conventional in the pattern of `*.subdomain.domain.tld`, whereas here it appears that the parts of the domain were swapped, resulting in the seemingly unconventional pattern of `subdomain.*.domain.tld`.

My apologies for the unsolicited comment here, especially if my understanding or interpretation is incorrect, but I thought saying something might provide an opportunity to make the appropriate changes if warranted.

Anyway, keep up the great work!

Reply:
Hey @brl0 that's a very valid point. I will fix that.